src/main/java/mate/academy/bookstore/repository/book/spec/AuthorSpecificationProvider.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.bookstore.repository.book.spec;
+
+import java.util.Arrays;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.repository.SpecificationProvider;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthorSpecificationProvider implements SpecificationProvider<Book> {
+    public Specification<Book> getSpecification(String[] params) {
+        return (root, query, criteriaBuilder)
+                -> root.get("author").in(Arrays.stream(params).toArray());
+    }
+
+    @Override
+    public String getKey() {
+        return "author";
```

Comment:
Better make key a public constant and you can use it in BookSpecificationBuilder(the same for other providers)

Reply:
Done